### PR TITLE
Scopes: Add Handler for returning dashboards related to a list of scopes. 

### PR DESCRIPTION
**Pull request by bergquist**
We should follow the same idea for the API that loads dashboards for a set of scopes.

### CODE REVIEW

File: pkg/registry/apis/scope/find_scoped_dashboards.go [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+}
+
+func (f *findScopedDashboardsREST) NewConnectOptions() (runtime.Object, bool, string) {
+	return nil, false, "" // true means you can use the trailing path as a variable
+}
+
+func (f *findScopedDashboardsREST) Connect(ctx context.Context, name string, opts runtime.Object, responder rest.Responder) (http.Handler, error) {
+	// See: /pkg/apiserver/builder/helper.go#L34
+	// The name is set with a rewriter hack
+	if name != "name" {
```
Comment (kylebrandt) [inline pkg/registry/apis/scope/find_scoped_dashboards.go:62]: This doesn't work currently as it has to match `/find$`

Comment (kylebrandt) [reply pkg/registry/apis/scope/find_scoped_dashboards.go:62]:
```
FindScopedDashboardsResults

GET
/apis/scope.grafana.app/v0alpha1/namespaces/{namespace}/find_scope_dashboard/{name}
```

and 

```
TreeResults


GET
/apis/scope.grafana.app/v0alpha1/namespaces/{namespace}/find
```

are inconsistent in that the URL for ScopedDashboards has name of what it is returning in the path, but `find` is for scopenodes, but doesn't have that in the path.

@ryantxu / @toddtreece Any thoughts on what a good pattern might be here to make these consistent?

Comment (kylebrandt) [reply pkg/registry/apis/scope/find_scoped_dashboards.go:62]: Maybe something like:

```
ScopedDashboardsResults
GET
/apis/scope.grafana.app/v0alpha1/namespaces/{namespace}/scoped_dashboard_results/find/{name}
and

TreeResults
GET
/apis/scope.grafana.app/v0alpha1/namespaces/{namespace}/tree_results/find/{name}
```

- This drops "Find" on the type name for one of them, uses underscore word separators, and has the separated return type for find before "find" in the url, and matches the current rewrite regex.

Comment (toddtreece) [reply pkg/registry/apis/scope/find_scoped_dashboards.go:62]: another couple options:
* `/apis/scope.grafana.app/v0alpha1/namespaces/{namespace}/views/node_tree`
* `/apis/scope.grafana.app/v0alpha1/namespaces/{namespace}/views/dashboards`

Comment (kylebrandt) [reply pkg/registry/apis/scope/find_scoped_dashboards.go:62]: I like the "views" prefix as that makes sense to me, but I'm also okay with find. This adds a view that is returning ScopeDashboardBinding , so I don't think we would want to call it dashboards., maybe dashboard_bindings.

**Comment by kylebrandt** [comment]
Is now like:

image

```
### Find scope dashboard bindings for scopes slothVoteTracker and slothClusterNorth (note the literal `name` is required)
GET {{baseURL}}/apis/scope.grafana.app/v0alpha1/namespaces/default/find/scope_dashboard_bindings?scope=slothVoteTracker&scope=slothClusterNorth
Accept: application/json
```

```
### Top Level (Forest)
GET {{baseURL}}/apis/scope.grafana.app/v0alpha1/namespaces/default/find/scopenodechildren
Accept: application/json


### Top Level Applications
GET {{baseURL}}/apis/scope.grafana.app/v0alpha1/namespaces/default/find/scopenodechildren?parent=applications
Accept: application/json

### Top Level Applications
GET {{baseURL}}/apis/scope.grafana.app/v0alpha1/namespaces/default/find/scopenodechildren?parent=
Accept: application/json
```

**Comment by bergquist** [comment]
👍 I like the new names :) But I cannot approve the PR since I'm the author.

**Review by kylebrandt — APPROVED** [review]

**Pull request merged**